app/components/publish-actions.tsx

```diff
@@ -78,14 +81,22 @@ const Embed = ({ chartWrapperRef, configKey, locale }: PublishActionProps) => {
 
   useEffect(() => {
     const { origin } = window.location;
-    setEmbedUrl(`${origin}/${locale}/embed/${configKey}`);
-    setEmbedAEMUrl(`${origin}/api/embed-aem-ext/${locale}/${configKey}`);
-  }, [configKey, locale]);
+    setEmbedUrl(
+      `${origin}/${locale}/embed/${configKey}${withoutBorder ? "?disableBorder=true" : ""}`
+    );
+    setEmbedAEMUrl(
+      `${origin}/api/embed-aem-ext/${locale}/${configKey}${withoutBorder ? "?disableBorder=true" : ""}`
+    );
+  }, [configKey, locale, withoutBorder]);
 
   const handleChange = useEvent((e: ChangeEvent<HTMLInputElement>) => {
     setIsResponsive(e.target.value === "responsive");
   });
 
+  const handleStylingChange = useEventCallback((checked: boolean) => {
+    setWithoutBorder(checked);
+  });
```

Comment:
Nit: As now we have two callbacks, I'd change the name of `handleChange` to `handleResponsiveChange`, now it looks quite generic. Also, there's `useEvent` and `useEventCallback` hooks used after each one, while they have the same purpose – I'd settle on using only one of them 👀

Reply:
Ah okay alright, tried to copy and example from an other instance using Switch